Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/mothership-computers.yml

```diff
@@ -0,0 +1,27 @@
+# The empress console
+- type: entity
+  id: EmpressMothershipComputer
+  name: Empress mothership console
+  parent: BaseMothershipComputer
+  components:
+  - type: Sprite
+    sprite: _NF/Structures/Machines/computers.rsi
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: shipyard_security
+    - map: ["computerLayerKeys"]
+      state: telesci_key
+  - type: ShipyardListing
+    name: Empress
+    shuttles:
+    - Interceptor
+    - Fighter
+    - Cleric
+    - Hospitaller
+    - Inquisitor
+    - Rogue
+    - Templar
```

Comment:
shuttles:
    - Fighter
    - Cleric
    - Rogue

No need for others on this one.